lib/msal-browser/src/crypto/BrowserCrypto.ts

```diff
@@ -0,0 +1,44 @@
+import { BrowserStringUtils } from "../utils/BrowserStringUtils";
+import { BrowserAuthError } from "../error/BrowserAuthError";
+
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const HASH_ALG = "SHA-256";
+
+export class BrowserCrypto {
+
+    constructor() {
+        if (!(window["msCrypto"] || window.crypto)) {
```

Comment:
I think you should create a helper function for this check.

Reply:
Done